src/__tests__/lib/exampleData.js

```diff
@@ -225,12 +225,12 @@ export const crossRealmBot: CrossRealmBot = makeCrossRealmBot({ name: 'bot' });
 
 const randStreamId: () => number = makeUniqueRandInt('stream IDs', 1000);
 export const makeStream = (
-  args: {| name?: string, description?: string |} = Object.freeze({}),
+  args: {| name?: string, description?: string, streamId?: number |} = Object.freeze({}),
```

Comment:
nit: Several of these functions that make data objects (`streamMessage`, `pmMessage`, etc.) accept the property names directly, without bothering to use camelCase. Best to do so here, for consistency, I think: so, `stream_id` instead of `streamId`.